feature-libs/product-configurator/rulebased/components/overview-sidebar/configurator-overview-sidebar.component.html

```diff
@@ -0,0 +1,16 @@
+<div class="cx-sticky-menu">
+  <div class="cx-menu-bar">
+    <button role="tab" [class.active]="!showFilter" (click)="onMenu()">
+      {{ 'configurator.overviewSidebar.menu' | cxTranslate }}
+    </button>
+    <button role="tab" [class.active]="showFilter" (click)="onFilter()">
+      {{ 'configurator.overviewSidebar.filter' | cxTranslate }}
+    </button>
+  </div>
+
+  <cx-configurator-overview-filter *ngIf="showFilter">
+  </cx-configurator-overview-filter>
+
+  <cx-configurator-overview-menu *ngIf="!showFilter">
+  </cx-configurator-overview-menu>
+</div>
```

Comment:
@AlexanderKuschner: There is no component test at all for this component. I would expect a component test that toggles filter and checks what is displayed.